docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,138 @@
+- `syntheticName:` The long name of the token
+- `syntheticSymbol:` The short name of the token or ticker symbol
+- `financialProductLibrary:` The financial library your contract will use to calculate the payment at expiry. For the list of the available financial product libraries we have available, you can go [here](https://github.com/UMAprotocol/protocol/tree/master/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries).
+
+## Deploying on Mumbai
+
+Before running this command, you should customize the parameters to your needs. YOUR_NODE_URL should be filled in with a url for the network that you wish to deploy to and the `LongShortPairCreator` value should be substituted with the creator address on that same network. Added to that you should also choose the corresponding financial product library (FPL) that makes sense for your use case. The script has been pre-filled with the FPL for the covered call options
+
+All the required Mumbai addresses can be seen on our [Github repo](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/80001.json) with the most important items in the list for specific launch are:
+
+- `Mumbai LongShortPairCreator` 
+- `BinaryOptionLongShortPairFinancialProductLibrary` 
+- `CoveredCallLongShortPairFinancialProductLibrary` 
+- `LinearLongShortPairFinancialProductLibrary` 
+- `RangeBondLongShortPairFinancialProductLibrary` 
+
+Before deploying, please note that the script will deploy to the network that your node URL is for. If using Infura for a Mumbai deployment, `YOUR_NODE_URL` will follow this format:
+
+
+```bash
+wss://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
+
+```bash
+https://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+Run the deployment script with your specific parameters.
+```bash
+node index.js --gasprice 1 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x6883FeB1c131F58C1Cd629289Da3dE0051d2aa0d --expirationTimestamp 1643678287 --collateralPerPair 1000000000000000000 --priceIdentifier ETHUSD --collateralToken 0xd0a1e359811322d97991e03f863a0c30c2cf029c --syntheticName "ETH 9000 USD Call [December 2021]" --syntheticSymbol ETHc9000-1221 --financialProductLibrary 0xc19B7EF43a6eBd393446F401d1eCFac01B181ac0
+```
+## Deploying to Polygon Mainnet
+
+The method to deploy onto mainnet is the exact same as with Mumbai, with a few key items to note. Your `url`, `lspCreatorAddress` and `financialProductLibrary` will need to be amended for use on Polygon mainnet. Since the library addresses are different between the networks. 
+
+All the required Polygon Mainnet addresses can be seen on our [Github repo](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json) with the most important items in the list for specific launch are:
+
+- `Polygon LongShortPairCreator` 
```

Comment:
```suggestion
- `LongShortPairCreator` 
```

Reply:
Actually I think it's worth removing just the specific contract listings since this is exactly the same as the mumbai listing.